packages/pluggableWidgets/popup-menu-web/src/__tests__/PopupMenu.spec.ts

```diff
@@ -48,7 +48,7 @@ describe("Popup menu", () => {
         it("triggers action", () => {
             basicItemProps.action = actionValue();
             const popupMenu = createPopupMenu(defaultProps);
-            const event: any = { target: {} };
+            const event: any = { target: {}, preventDefault: jest.fn(), stopPropagation: jest.fn() };
```

Comment:
Can we also verify that these are actually called in a "on click" test?

Reply:
Fixed and also fixed the same for custom items